projects/capybara/baseline/keras/baseline_utils.py

```diff
@@ -0,0 +1,427 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2017, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+import csv
+import copy
+import itertools
+import numpy as np
+import pandas as pd
+import os
+
+from sklearn.metrics import accuracy_score
+
+from keras.utils import np_utils
+from keras.models import Sequential
+from keras.layers.core import Dense
+from keras.utils.visualize_util import plot
```

Comment:
@ywcui1990 I removed the keras model visualization. It was a but superfluous since the network is really simple. You should not encounter this error anymore.